sdp/processors/datasets/mtedx/create_initial_manifest.py

```diff
@@ -0,0 +1,83 @@
+import os
+from pathlib import Path
+from typing import List
+import librosa
+from sdp.processors.base_processor import BaseParallelProcessor, DataEntry
+from sdp.utils.common import download_file, extract_archive
+
+MTEDX_URL = "https://www.openslr.org/resources/100/mtedx_{language_id}.tgz"
+
+class CreateInitialManifestMTEDX(BaseParallelProcessor):
```

Comment:
Please add this processor to the list of "Dataset-specific processors" in the docs

Reply:
Done